www/includes/easyparliament/page.php

```diff
@@ -1287,13 +389,33 @@ public function error_message($message, $fatal = false, $status = 500) {
             );
         }
 
-        $this->message($message, 'error');
+        // if the page has started then we're most likely in an old school page
+        // so we should just print out the error, otherwise stick it in the error
+        // global which will then be displayed by the header template
+        if ( $this->page_started() ) {
+            $this->message($message, 'error');
+        } else {
+            if ( !isset($page_errors) ) {
+                $page_errors = array();
+            }
+            $page_errors[]  = $message;
+        }
 
         if ($fatal) {
+            if (!$this->page_started()) {
+                if (!headers_sent()) {
+                    header("HTTP/1.0 $status Internal Server Error");
+                }
+                $this->page_start();
+            }
+
+            $this->message($message, 'error');
```

Comment:
I think this line isn't needed, as if the page has been started, it was output above, and if it hasn't, it'll be output by `page_start`.